modules/alloy/src/main/ts/ephox/alloy/ui/types/SliderTypes.ts

```diff
@@ -17,6 +17,7 @@ export interface SliderValueXY {
   readonly y: number;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-duplicate-type-constituents
```

Comment:
Is this something we should try to resolve in some other follow up ticket?

Reply:
I am too worried about this one as I think for clarity having `SlideX | SliderY | SliderXY` is less confusing than `SliderX | SliderXY`